index.js

```diff
@@ -98,6 +98,8 @@ module.exports = {
 
     merge(pkg, additions);
 
+    pkg.description = `Test app for ${this.locals(this.options).addonName} addon`;
```

Comment:
just a minor tweak, indeed subjective, however should be more helpful than "Small description for test-app goes here"